utilities/oadp.py

```
@@ -28,10 +36,49 @@
+            Kubernetes dynamic client used to locate the Velero pod.
+
+    Raises:
+        ResourceNotFoundError:
+            If the Velero Pod cannot be found.
+        Exception:
+            Propagates any exception raised during deletion after logging context.
+    """
+    try:
+        velero_pod = get_pod_by_name_prefix(client=client, pod_prefix="velero", namespace=ADP_NAMESPACE)
+        if velero_pod is None:
+            raise ResourceNotFoundError("Velero pod not found in namespace openshift-adp")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)